ui/src/app/views/workspace-list/component.tsx

```diff
@@ -172,7 +172,7 @@ export class WorkspaceCard extends React.Component<
                   {wp.workspace.name}</div>
               </Clickable>
             </div>
-            <div style={styles.workspaceDescription}>{wp.workspace.description}</div>
+            {/*<div style={styles.workspaceDescription}>{wp.workspace.description}</div>*/}
```

Comment:
I'd just delete this if we don't need it. But it should probably be a different one of those text fields? Maybe `scientificQuestion`? This is a good Karthik question.

Reply:
I believe that RW-2594 answers this